src/scripts/audit-events.js

```diff
@@ -1,6 +1,10 @@
-const { getSvelteFiles, componentDetails } = require('./common')
-const path = require('path')
-const fs = require('fs/promises')
+import { getSvelteFiles, componentDetails } from './common.js'
+import path from 'path'
+import fs from 'fs/promises'
+import { fileURLToPath } from 'url'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
```

Comment:
I feel like as these aren't the actual metadata variables we should just give them normal names:
```suggestion
const filename = fileURLToPath(import.meta.url)
const dirname = path.dirname(__filename)
```